sunbeam-python/sunbeam/provider/maas/steps.py

```diff
@@ -1831,6 +1831,23 @@ def _get_loadbalancer_range(self) -> str | None:
         """Return loadbalancer range from public space."""
         return self.ranges
 
+    def _get_loadbalancer_l2_interfaces(self) -> str | None:
+        """Return l2 interfaces to use for loadbalancer.
+
+        For maas mode, the interfaces are corresponding to infra,
+        internal and public spaces on any one of control nodes.
+        """
+        spaces = self.deployment.get_spaces(
+            [Networks.MANAGEMENT, Networks.INTERNAL, Networks.PUBLIC]
```

Comment:
I don't get why you would want the ifnames from internal and public. I believe L2 annoucement should only come from MANAGEMENT? Or does that allow for better routing?

Reply:
You are right, for announcements Management should be good enough.